snippets/numberWithCommas.md

```diff
@@ -0,0 +1,19 @@
+### numberWithCommas
+
+Use commas to separate digits。
```

Comment:
Change this line to: 

Convert a float-point arithmetic to the [Decimal mark](https://en.wikipedia.org/wiki/Decimal_mark) form.